plugins/env/app/views/samson_env/_fields.html.erb

```diff
@@ -5,9 +5,11 @@
   </legend>
 
   <% if ENV['DEPLOYMENT_ENV_REPO'] %>
-    <%= form.input :use_env_repo, label: "Use GitHub repository #{ENV['DEPLOYMENT_ENV_REPO']} to manage deployment environment variables",
-                   as: :check_box
-    %>
+    <% project = form.object %>
+    <%= check_box_tag(:use_env_repo) %>
+    <% link_target =  "#{ENV['DEPLOYMENT_ENV_REPO']}/#{project.permalink}.env.erb" %>
+    <% label_text = "Use the GitHub repo for deployment env vars for this project here #{link_target}" %>
+    <%= label_tag(:use_env_repo, link_to(label_text,link_target)) %>
```

Comment:
`label:` with html_safe should also work

Reply:
Done